generators/server/templates/src/main/java/package/domain/_User.java

```diff
@@ -183,7 +184,7 @@ public String getLogin() {
 
     //Lowercase the login before saving it in database
     public void setLogin(String login) {
-        this.login = login.toLowerCase(Locale.ENGLISH);
+        this.login = login == null ? null : login.toLowerCase(Locale.ENGLISH);
```

Comment:
yes, this makes our Sonar build fail :-(
I also don't think this code is easy to read